lua/starfall/libs_sh/hologram.lua

```diff
@@ -293,14 +293,33 @@ else
 		end
 	end
 
+	local render_GetColorModulation, render_GetBlend = render.GetColorModulation, render.GetBlend
+	local render_SetColorModulation, render_SetBlend = render.SetColorModulation, render.SetBlend
+
 	--- Manually draws a hologram, requires a 3d render context
 	-- @client
-	function hologram_methods:draw()
+	-- @param bool? tint If true, renders the hologram with it's color and opacity. This is the default behavior.
+	function hologram_methods:draw(tint)
```

Comment:
Not a big fan of '`true` by default' params. Maybe just invert it to `noTint` if you really want it as the default behavior?

```diff
diff --git a/lua/starfall/libs_sh/hologram.lua b/lua/starfall/libs_sh/hologram.lua
index e37989bc..48140fe9 100644
--- a/lua/starfall/libs_sh/hologram.lua
+++ b/lua/starfall/libs_sh/hologram.lua
@@ -298,14 +298,16 @@ else
 
 	--- Manually draws a hologram, requires a 3d render context
 	-- @client
-	-- @param bool? tint If true, renders the hologram with it's color and opacity. This is the default behavior.
-	function hologram_methods:draw(tint)
+	-- @param bool? tint Bla bla bla
+	function hologram_methods:draw(noTint)
 		if not instance.data.render.isRendering then SF.Throw("Not in rendering hook.", 2) end
 			
 		local holo = getholo(self)
 		holo:SetupBones()
 
-		if tint ~= false then
+		if noTint then
+			holo:DrawModel()
+		else
 			local cr, cg, cb, ca = holo:GetColor4Part()
 			local ocr, ocg, ocb = render_GetColorModulation()
 			local oca = render_GetBlend()
@@ -317,8 +319,6 @@ else
 				
 			render_SetColorModulation(ocr, ocg, ocb)
 			render_SetBlend(oca)
-		else
-			holo:DrawModel()
 		end
 	end
 end
```